samples/cpp/benchmark_app/main.cpp

```diff
@@ -262,6 +262,11 @@ int main(int argc, char* argv[]) {
             config.erase(device);
         }
 
+        // If set batch size, disable the auto batching
+        if (FLAGS_b > 0) {
```

Comment:
I suppose we need to move these lines below, to set it after loaded config to have a first priority (see how it is done for cache_dir).

Reply:
I think it doesn't matter, because it should have been handed in the implement of this property. It's priority is lower than the property of load model.

Reply:
> I think it doesn't matter, because it should have been handed in the implement of this property. It's priority is lower than the property of load model.

But config is set using core set_property method as well:

https://github.com/openvinotoolkit/openvino/blob/4973a2976ea93b4c687bf7b4263974d36fae4266/samples/cpp/benchmark_app/main.cpp#L426

Reply:
core.set_property(ov::hint::allow_auto_batching(false)); only store a flag and will only be used if local property is not set.
https://github.com/RICKIE777/openvino/blob/4973a2976ea93b4c687bf7b4263974d36fae4266/src/inference/src/ie_core.cpp#L758

Reply:
@RICKIE777  could you move L265-268 to L343?